public/data/connections.yaml

```diff
@@ -1145,6 +1145,19 @@ supported_databases:
   va_supported: true
   classification_supported: true
   notes: Classification via GDP and ISDC
+  network_traffic: K-TAP
```

Comment:
These are all supported features for the S-TAP (Software TAP).  When someone selects the S-TAP method, then they'll see these types of details.